include/transport/transport.h

```diff
@@ -74,8 +74,8 @@ struct TransportRemote
  */
 struct TransportConfig
 {
-  char *tls_cert_filename;                        /// QUIC TLS certificate to use
-  char *tls_key_filename;                         /// QUIC TLS private key to use
+  const char *tls_cert_filename;                  /// QUIC TLS certificate to use
```

Comment:
Challenge is that `TransportConfig` is mutable context/config. const values prevent that.  Changing
this will require implementations to update, such as libquicr. 

I would recommend though that we switch this to string.